docs/guide/styles.md

```
@@ -0,0 +1,83 @@
+```
+
+For convenience, I collected my frequent used styles in a package called [respo-ui](https://github.com/Respo/respo-ui).
+You can find more in the [source code][styles].
+
+[styles]: https://github.com/Respo/respo-ui/blob/master/src/respo_ui/style.cljc
+
+### Static Styles
+
+A macro `respo.css/defstyle` has been added for add `<style>...</style>` referred with `:class-name`. It's less dynamic, which means you cannot pass parameters to styles in this way. It will insert into `<head>...</head>` a `<style>...</style>` element. It runs before `main!` and `reload!`.
+
+![](https://user-images.githubusercontent.com/449224/165848778-5fb6e192-e1f3-426d-b387-e4ab7cb120c5.png)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)